.github/ISSUE_TEMPLATE/Team_handbook.md

```diff
@@ -0,0 +1,45 @@
+---
+name: Team handbook
+about: Propose a change to the team's handbook.
+
+---
+
+- [ ] The issue's title has a prefix of `[Team handbook]`.
+- [ ] The issue has been labeled with `Team handbook`.
+- [ ] There is more than one variant in the proposal.
+
+A summary of the goal of this proposal.
+
+# Variants
+
+## Variant 1: A single-sentence description of this variant.
+
+A longer-form description of the variant.
+
+Pros:
+
+- Individual items describing the positive aspects of this variant.
+
+Cons:
+
+- Individual items describing the negative aspects of this variant.
+
+Action items:
+
+- A list of actions that will have to be taken in order to make this handbook change.
+
+## Variant 2: A single-sentence description of this variant.
```

Comment:
What is the difference between the two variants? It looked like there was one but now there's not

Reply:
Ah, hm. This is just meant to be an example. Let me try again...

Reply:
Updated.